terraform/prod-app.tfvars

```diff
@@ -0,0 +1,10 @@
+postgres_config = {
+  instance_class = "db.t3.small"
+  identifier     = "gpg-prod-db"
+  db_name        = "gpgProdDb"
+  username       = "postgres"
+  password       = "postgres"
```

Comment:
For my peace of mind, is there a future ticket to change this so we're not storing our database credentials in plain text, in the code? If so it would be nice to have a todo comment here referencing it.

Reply:
Yes, I've moved them as environment variables in devops.